dm/loader/util.go

```diff
@@ -141,15 +144,61 @@ func getMydumpMetadata(cli *clientv3.Client, cfg *config.SubTaskConfig, workerNa
 	return loc.Position.String(), loc.GTIDSetStr(), nil
 }
 
+// getMydumpMetadataByExternalStorage gets Metadata by ExternalStorage.
+func getMydumpMetadataByExternalStorage(ctx context.Context, cli *clientv3.Client, cfg *config.SubTaskConfig, workerName string) (string, string, error) {
```

Comment:
`exstorage` package can handle local files, why not just enchancing existing `getMydumpMetadata`

Reply:
yes, it's better, i will try.